en/products/monitor/landing.ftl

```diff
@@ -0,0 +1,58 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/products/monitor/
+
+# HTML page title
+monitor-landing-page-title = { -brand-name-mozilla-monitor } — Protect Your Personal Data from Exposures
+
+# HTML page description
+monitor-landing-page-desc = Discover if your personal information, from phone numbers to home addresses, has been exposed online. Join 10 million users worldwide in reclaiming your privacy. Start with a free scan.
+
+monitor-hero-heading = Find out where your private info is exposed — and take it back
+monitor-hero-tagline = We scan data breaches to see if your data has been leaked and give you steps to fix it.
+
+## Get free scan form
+
+monitor-form-enter-email-to-scan = Enter email to scan here
+monitor-form-get-free-scan = Get free scan
+monitor-form-enter-your-email-address = Enter your email address to check for data breach exposures.
+
+##
+
+monitor-data-breaches-happen-every = <strong>Data breaches</strong> happen every 11 minutes, exposing your private information — but don’t worry, we can help.
+monitor-help-fix-exposures = We’ll help fix your exposures
+
+# Variables:
+#   $privacy (url) - https://www.mozilla.org/privacy/
+monitor-our-mission-is = Our mission is to put control of your personal data back in your hands. We’ll help you resolve data breaches and keep your info private – and we’ll <a href="{ $privacy }">respect your privacy</a> in the process.
```

Comment:
I think (monospace font's doesn't help), that this is using the wrong dash: 
* This string: –
* Other strings (correct): —